lib/mix/tasks/swagger.generate.ex

```diff
@@ -131,22 +131,19 @@ defmodule Mix.Tasks.Phoenix.Swagger.Generate do
     case function_exported?(Mix.Project.get, :swagger_info, 0) do
       true ->
         info = Mix.Project.get.swagger_info
-        title = Keyword.get(info, :title, nil)
-        version = Keyword.get(info, :version, nil)
+
+        # :title and :version are mandatory fields,
+        # so we need to add default values
+        # if they are not exists
+        title = Keyword.get(info, :title, @default_title)
+        version = Keyword.get(info, :version, @default_version)
+
         # collect :info swagger fields to the map
         info = List.foldl(info, %{},
           fn ({info_key, info_val}, map) ->
             Map.put_new(map, info_key, info_val)
           end)
-        # :title and :version are mandatory fields,
-        # so we need to check and add default values
-        # if they are not exists
-        info = if title == nil do
-          Map.put_new(info, :title, @default_title)
```

Comment:
I have an issue whit this, because I put a value in the title and version and with this logic always put something else (at this time dind't remember the result or maybe I was using in a bad way). Using the default_option in `Keyword.get` with default_variable allow us to remove this if and work in the same way. Maybe it's the way that elixir new version works, but if my commit breaks the result I'm open to rollback.